students/matiasli/lesson02/all_answers.py

```diff
@@ -0,0 +1,45 @@
+# all answers 
+
+
+x =  '+ - - - - + - - - - +'
+y = '|         |         |'
+for i1 in range(2):
+	print(x)
+	for i2 in range(4):
```

Comment:
the indentation has issues throughout the files - should be exactly 4 spaces (no tabs); please adjust your editor